UPDATING.md

```diff
@@ -23,6 +23,9 @@ assists people when migrating to a new version.
 
 ## Next Version
 
+* [7667](https://github.com/apache/incubator-superset/pull/7667): a change to
```

Comment:
nit: remove the "to" before reference 

`a change to make Unix (epoch) timestamps reference UTC as opposed to the local time zone.`